src/lib/containers/RssFeedCards.tsx

```diff
@@ -151,15 +140,15 @@ export default class RssFeedCards extends React.Component<RssFeedCardsProps, Rss
             })}
           </div>
           {this.state.rssFeed.items &&
-            this.state.rssFeed.items.length > this.props.defaultItemsToShow &&
-            !this.state.isShowingMoreItems && (
+            this.state.rssFeed.items.length > this.props.itemsToShow && (
               <div className="RssFeedViewAllNewsButtonContainer">
-                <button
-                  className="btn homepage-button"
-                  onClick={this.onClickShowMoreItems()}
-                >
-                  VIEW ALL NEWS
-                </button>
+                <a href={this.state.rssFeed.link} target="_blank">
```

Comment:
code scanning wants rel="noopener/noreferrer" on the anchor tag, could you add that?